src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
```

Comment:
Use 'const' for 'finalPrice' since it's not being reassigned, but rather calculated once based on the condition.